modules/ROOT/pages/ibm/ibm-mq-xml-ref.adoc

```diff
@@ -1,11 +1,11 @@
-= IBM MQ Connector Reference
+= IBM MQ Connector Reference - Mule 4
```

Comment:
There are some sections where some entries have periods and others don't. I don't know if you care. 

There are typos in text I can't edit, so I assume the text is coming in from somewhere else. Here's one:

"Defines which Cipher Suite Mapping to use. Oracle JDK or IBM JDK. Configure depending on the Host JDK. More information." The first period should be a colon.